pkg/apply/layerApplier.go

```diff
@@ -114,7 +114,7 @@ func getLabel(hr metav1.ObjectMeta) string {
 func removeResourceVersion(obj runtime.Object) string {
```

Comment:
see https://github.com/fidelity/kraan/pull/184/files#diff-9126395a3abfddd5d452095ceff4984434148f8370530962de3aef92d4b08202L114 I fixed this by removing the returned value, it is not actually used